apps/network-alerts/test/unit/attention/notify_test.js

```diff
@@ -0,0 +1,244 @@
+/*global
+   AudioContext,
+   MockAudioContext,
+   MockNavigatorSettings,
+   MocksHelper,
+   Notify
+*/
+
+'use strict';
+
+require('/shared/test/unit/mocks/mock_navigator_moz_settings.js');
+require('/shared/test/unit/mocks/mock_audio.js');
+require('/js/attention/notify.js');
+
+var mocksHelperNotifications = new MocksHelper([
+  'AudioContext'
+]).init();
+
+suite('check the ringtone and vibrate function', function() {
+  var realMozSettings;
+  var getStub, setStub;
+  var isDocumentHidden = true;
+
+  mocksHelperNotifications.attachTestHelpers();
+
+  function MockOscillatorNode() {
```

Comment:
just idea, maybe we can leverage sinon more and get rid of all this Mock classes? Just like this:
```
this.sinon.stub(
  MockAudioContext.prototype, 'createOscillator',
  () => sinon.stub({
    frequency: { value: 0 },
    type: '',
    connect() {},
    start() {},
    stop() {}
  })
);

this.sinon.stub(
  MockAudioContext.prototype, 'createGain',
  () => sinon.stub({
    gain: sinon.stub({ setValueCurveAtTime() {} }),
    connect() {}
  })
);
```